create_pdfs.py

```diff
@@ -81,13 +85,31 @@ def main():
     certificate_data = []
 
     if args.course_id:
-        course_list = [args.course_id]
+        course_id = args.course_id
+        # Default course name to course_id when not supplied
```

Comment:
Is this necessary?
The code itself says "long course should use the passed in arg, but it can use the course_id if it's not set".
It's not clear to me that the comment is saying anything that the code already doesn't